app/src/main/java/ch/epfl/favo/favor/Favor.java

```diff
@@ -20,6 +19,7 @@
   public enum Status {
```

Comment:
Im wondering if there is a better way to handle temporary states then adding it to enum Status class. These states are stored in the database so its not a good idea to be storing temporary states for favors in our backend.

Reply:
Yeah good point. I created another enum holding temporary status for the views